src/resolvers/Mutation/auth.js

```diff
@@ -0,0 +1,48 @@
+const { google } = require('../../services/google');
+const jwt = require('jsonwebtoken');
+
+// Helpers.
+
+async function getPrismaUser({ db }, googleId) {
```

Comment:
Just for reference:
you can make use the arrow function declaration to make the code more declarative:
ex `cont getPrismaUser = async ({ db }, googleId) => {}`
It's ok for current syntax.